### DISCUSSION TITLE
Where the parameter `Visibility` come from?

### DISCUSSION THREAD

**Discussion by AdenJustice** [oUF-wow/oUF#704] [Q&A] [answered]
https://github.com/oUF-wow/oUF/blob/91d0de9107afeb22f66d39d3218fbb3d07f1341c/elements/classpower.lua#L196
https://github.com/oUF-wow/oUF/blob/91d0de9107afeb22f66d39d3218fbb3d07f1341c/elements/classpower.lua#L200

`Visibility` in `Visibility` seems wrong...?

**Comment by p3lim** [comment, marked as answer]
No that's correct.
